### PR TITLE
Converted Elemental Attribute Damage Adjustment Tables to YAML

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: N/A

* **Server Mode**: Both

* **Description of Pull Request**: 

  * Converts the Elemental Attribute Damage Adjustment Tables file into YAML.
  * Includes CSV2YAML converter.

### CODE REVIEW

File: src/map/status.hpp [hunk 1]
```diff
@@ -116,6 +118,26 @@ class SizeFixDatabase : public TypesafeYamlDatabase<int32, s_sizefix_db> {
...
+	int16 attr_fix_table[MAX_ELE_LEVEL][ELE_MAX][ELE_MAX];
+
+public:
+	AttributeDatabase() : YamlDatabase("ATTRIBUTE_DB", 1) {
+
+	}
+
+	void clear() { 
+		std::fill_n(&attr_fix_table[0][0][0], MAX_ELE_LEVEL * ELE_MAX * ELE_MAX, 100);
+	};
```
Comment (Lemongrass3110) [inline src/map/status.hpp:132]:
```suggestion
	}
```

File: src/map/status.hpp [hunk 2]
```diff
@@ -116,6 +118,26 @@ class SizeFixDatabase : public TypesafeYamlDatabase<int32, s_sizefix_db> {
 
 extern SizeFixDatabase size_fix_db;
 
+class AttributeDatabase : public YamlDatabase {
+private:
+	int16 attr_fix_table[MAX_ELE_LEVEL][ELE_MAX][ELE_MAX];
+
+public:
+	AttributeDatabase() : YamlDatabase("ATTRIBUTE_DB", 1) {
+
```
Comment (Lemongrass3110) [inline src/map/status.hpp:127]:
```suggestion
		this->clear();
```

File: src/map/status.cpp [hunk 3]
```diff
@@ -16048,6 +16044,7 @@ int status_readdb( bool reload ){
 		size_fix_db.load();
 		refine_db.load();
 	}
+	elemental_attribute_db.reload();	// fill the table with clear
```
Comment (Lemongrass3110) [inline src/map/status.cpp:16047]:
```suggestion
	elemental_attribute_db.load();
```

File: src/map/map.hpp [hunk 4]
```diff
@@ -344,6 +344,19 @@ enum e_element : int8{
 	ELE_RANDOM,
 };
 
+static std::unordered_map<std::string, e_element> um_eleid2elename {
```
Comment (Lemongrass3110) [inline src/map/map.hpp:347]: Why "static"?

**Review by Lemongrass3110 — APPROVED** [review]

File: src/map/map.hpp (hunk 4 above)
Comment (jasonch35) [reply src/map/map.hpp:347]: hi @Lemongrass3110 , im using debian, im getting error with this line, 

Screenshot 2021-08-21 221823

i think it has something to do with the string not being defined.

**Pull request merged**